examples/credential-registry/src/lib.rs

```diff
@@ -96,13 +91,12 @@ pub struct CredentialEntry<S: HasStateApi> {
     revocation_nonce: u64,
     /// Revocation flag
     revoked:          bool,
-    /// Schema reference and credential metadata.
-    /// This data is only needed when credential info is requested. In other
-    /// operations, `StateBox` defers loading credential data.
-    credential_data:  StateBox<CredentialData, S>,
+    /// Metadata URL of the credential (not to be confused with the metadata URL
+    /// of the **issuer**).
+    metadata_url:     MetadataUrl,
```

Comment:
We can put the `MetaDataUrl` in a StateBox as well, so it is lazy loaded.

Reply:
Ok, I'll wrap it in `StateBox`

Reply:
Done.